src/functions-templates/typescript/json/{{name}}.ts

```diff
@@ -1,5 +1,5 @@
 import type { Context } from "https://edge.netlify.com";
 
 export default async (request: Request, context: Context) => {
-  return context.json({ hello: "world" });
+  return Response.json({ hello: "world", location: context.geo.city });
```

Comment:
I added the `location` field here to be in line with https://github.com/netlify/cli/pull/5340#discussion_r1059948889.